docs/proposals/20201125-jaeger-traces-store.md

```diff
@@ -0,0 +1,122 @@
+# Jaeger traces store
+
+* **Owners:**:
+    * `@pavolloffay`
+
+* **Related Tickets:**
+    * [observatorium/issues/442](https://github.com/observatorium/observatorium/issues/442)
+    * Depends on: [Traces ingestion API and OpenTelemetry collector](https://github.com/observatorium/observatorium/pull/443)
+
+## Table of Contents
+
+- [TLDR](#tldr)
+- [Why](#why)
+- [Goals](#goals)
+- [Non-Goals](#non-goals)
+- [How](#how)
+    * [Jaeger configuration](#jaeger-configuration)
+        + [Single Jaeger instance](#single-jaeger-instance)
+            - [OpenTelemetry collector configuration](#opentelemetry-collector-configuration)
+        + [Jaeger instance per tenant](#jaeger-instance-per-tenant)
+            - [OpenTelemetry collector configuration](#opentelemetry-collector-configuration)
+    * [Jaeger Query](#jaeger-query)
+        + [Expose only Jaeger Query API](#expose-only-jaeger-query-api)
+        + [Expose Jaeger Query UI](#expose-jaeger-ui)
+- [Action Plan](#action-plan)
+- [References](#references)
+
+<small>
+
+<i>
+
+<a href="http://ecotrust-canada.github.io/markdown-toc/">
+Table of contents generated with markdown-toc
+</a>
+
+</i>
+
+</small>
+
+## TLDR
+
+This proposal adds Jaeger deployment with a persistent storage (e.g. Elasticsearch) to store trace data and it as well defines a query API to search for traces.
+
+## Why
+
+Traces store adds a crucial capability to Observatorium to persist and query tracing data.
+
+## Goals
+
+* Configure OpenTelemetry collector to export data to Jaeger collector.
+* Deploy Jaeger with a persistent storage.
+* Define query API for trace data.
+
+## Non-Goals
+
+* Define trace ingestion API - it is done in [Traces ingestion API and OpenTelemetry collector](https://github.com/observatorium/observatorium/pull/443) proposal.
+
+## How
+
+The how is split into the following sections:
+* Jaeger configuration - describes how Jaeger components are deployed.
+* Jaeger Query - describes how users access data stored in Jaeger.
+
+### Jaeger configuration
+
+Currently, the most adopted and supported storage in Jaeger is Elasticsearch. Red Hat also supports Jaeger with Elasticsearch (6.x) as part of the OpenShift product. Therefore, this proposal assumes Elasticsearch will be used as a storage backend. Distributed tracing team at Red Hat is looking at alternative storages hence the storage technology can change in the future.
+
+Follows possible multitenant Jaeger deployment topologies. All deployment topologies assume that Jaeger will be deployed by the [Jaeger Operator](https://github.com/jaegertracing/jaeger-operator) which depends on [OpenShift Elasticserach operator](https://github.com/openshift/elasticsearch-operator) and [Strimzi operator](https://operatorhub.io/operator/strimzi-kafka-operator) if Kafka is used. Jaeger can be as well configured to use externally managed Elasticsearch or Kafka instances.
```

Comment:
I am bit confused here. So we plan to use Elastic as storage or Kafka AND elastic  or just Kafka or just Elastic?